content/issues/63.markdown

```diff
@@ -0,0 +1,32 @@
+---
+title: Issue 63
+published: 2017-07-13
+---
+
+Welcome to another issue of Haskell Weekly!
+Haskell is a purely functional programming language that focuses on robustness, concision, and correctness.
+This is a weekly summary of what's going on in its community.
+
+## Featured
+
+todo
+
+## Package of the week
+
+todo
+
+## Call for Participation
+
+Are you interested in contributing to open-source Haskell projects but not sure where to start?
+Here are some tasks from the Haskell community for you to pick and get started!
+
+* [stack: "stack new" can't be pointed at an intranet site](https://github.com/commercialhaskell/stack/issues/2804)
```

Comment:
I know the title uses quotes, but I'd prefer backticks here since `stack new` is a command.